### ISSUE TITLE
Inject code at location doesn't work properly

### ISSUE THREAD

**Issue by phanirithvij** [TeamCodeStream/codestream#406]
codestream : v10.5.1

vscode: 
```
Version: 1.53.2 (user setup)
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:48:04.245Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
```

Inject injects at the wrong location.

This is the specific issue.
https://api.codestream.com/c/YC5QZ0N_FAF11qfv/1R_BWxbJQyaeIfZS4hbsWg

Screen record

https://user-images.githubusercontent.com/29627898/108362664-0abff300-721a-11eb-9b01-1058d3e527b7.mp4

**Comment by planteater** [comment]
Thanks for the detailed report @phanirithvij! We'll get this fixed asap!

**Comment by planteater** [comment]
@phanirithvij - This issue has been fixed in v10.7.0. Thanks again for the report!
